docs/tools/checkout/README.md

```diff
@@ -0,0 +1,42 @@
+---
+id: Checkout
+title: Checkout
+description: >-
+  Explainer on the what the checkout endpoint is which exists as part of
+  the Unlock Protocol app.
+---
+
+# Checkout
+
+Unlock Protocol has an app, most people are familiar with the Dashboard, Members
+and Keychain portions, but that app has an endpoint called checkout which accepts
+a configuration object as params and returns a custom configured checkout experience.
+
+## The experience
+This experience can also be used in a few different ways:
+1. To implement an Unlock enabled sign-in with Ethereum ([SIWE](../sign-in-with-ethereum/))
```

Comment:
Even though on our end this is the same "path" I would not mention it here as I think it may confuse people more...